module/Finna/src/Finna/Feed/LinkedEvents.php

```diff
@@ -203,6 +203,7 @@ public function getEvents($params)
                 $map = ['en-gb' => 'en'];
                 $this->language
                     = $map[$paramArray['language']] ?? $paramArray['language'];
+                $paramArray['language'] = $this->language;
```

Comment:
Koko logiikka on vähän pielessä. Miksi language:n tarvitsee olla mukana query:ssa? Eikö se ole aina sama kuin käyttäjän kieli, jolloin $this->language on jo se? 
Nykyisellään jos paramArray:ssa ei tulisi languagea, ei $this->language:akaan koskaan mapattaisi oikein (ts. jos siinä olisi en-gb, niin se ei muutu en:ksi) eikä lisättäisi paramArray:yn.

Voisi siirtää mappauksen constructoriin, poistaa language-parametrin ja lisätä languagen vain paramArrayyn tässä.

Reply:
Juu. Näin on hyvä.